webportal/webapp/src/app/account/loginsync/services/loginsyncservice.service.ts

```diff
@@ -9,8 +9,11 @@ import { CognitoService } from '../../../../services/cognito.service';
 export class LoginSyncService {
   httpOptions = {};
   env = environment.production ? 'prod' : 'dev';
-  linkAccountUrl = `${environment.LOGIN_GOV_ACCOUNT_LINK_URL}/${this.env}/${this.env}-link-account`;
-  accountLinkedUrl = `${environment.LOGIN_GOV_ACCOUNT_LINK_URL}/${this.env}/${this.env}-account-linked`;
+  // TODO: Additional problem here in that the current account API Gateway terraforms also add a suffix
+  // Omitting the redundant environment prefix in our paths would also make this less brittle
+  // Pulling account-link-env into the base URL would also clean this up
+  linkAccountUrl = `${environment.LOGIN_GOV_ACCOUNT_LINK_URL}/account-link-${this.env}/${this.env}-link-account-private`;
+  accountLinkedUrl = `${environment.LOGIN_GOV_ACCOUNT_LINK_URL}/account-link-${this.env}/${this.env}-account-linked-private`;
```

Comment:
@aaronFranssell Any thoughts on this? It feels a bit messy. Some of my thoughts:
I'd prefer if these lines looked more like this:
```
linkAccountUrl = `${environment.LOGIN_GOV_ACCOUNT_LINK_URL}/link-account`;
```
Changing the base URL to include the `account-link` path wouldn't be difficult, but in order to avoid the name redundancy we'd need to modify our API terraforms.

Reply:
I see what you mean... but I think that modifying Terraform would be more trouble than its worth.